account_invoice_repair_link/migrations/12.0.1.0.0/post-migration.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2019 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+
+def migrate(env, version):
```

Comment:
This should go in an init hook, not a migration, otherwise it will never be triggered

Reply:
Well, I don't think that you need this anyway. Because the invoice id is already in the repair order. The one2many does not store data in the model where it is used, but in the related model

Reply:
You're right, fixed!